### PR TITLE
Update Access Graph Docs

### PR DESCRIPTION
**Pull request by jakule**
Update Access Graph docs to reflect the latest changes and the new UI. 

Closes https://github.com/gravitational/access-graph/issues/551

### CODE REVIEW

File: docs/pages/access-controls/access-graph.mdx [hunk 1]
```diff
@@ -23,174 +23,171 @@ under the Permission Management section.
...
 
-To remove the graph node type, expand the node type in the sidebar and click on the
-remove button.
+![Show Access Path](../../img/access-graph/show-access-path.gif)
 
-![Remove node type](../../img/access-graph/remove-node-type.png)
+## Navigation
 
-### Expanding/collapsing graph nodes
+![Navigation](../../img/access-graph/sidebar.png)
```
Comment (stevenGravy) [inline docs/pages/access-controls/access-graph.mdx:119]: Would just make a cleaner version of this graphic

**Review by stevenGravy — APPROVED** [review]
Minor nit on graphic. looking good

File: docs/pages/access-controls/access-graph.mdx (hunk 1 above)
Comment (jakule) [reply docs/pages/access-controls/access-graph.mdx:119]: Updated https://github.com/gravitational/teleport/pull/41307/commits/bc473db2e79bccd6e8166219676bf6ed5aca523e

**Comment by jakule** [comment]
Friendly ping @r0mant @zmb3 @ptgott

File: docs/pages/access-controls/access-graph.mdx [hunk 2]
```diff
@@ -23,174 +23,171 @@ under the Permission Management section.
...
-2. User Groups
+![Identity Node](../../img/access-graph/identity-node.png)
 
-![User Group Node](../../img/access-graph/user-group-node.png)
+Identities are the actors that can access your infrastructure. They can be employees,
+contractors, machines or bots.
 
-User Groups are collections of users. They can be used to organize users
-based on their role or team.
+The number on the right hand side show "standing privileges".
```
Comment (r0mant) [inline docs/pages/access-controls/access-graph.mdx:35]:
```suggestion
The number on the right hand side shows "standing privileges".
```

**Review by r0mant — APPROVED** [review]

**Pull request merged**